### PR TITLE
[16.0][MIG] sale_coupon_incompatibility>sale_loyalty_incompatibility: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
Supersedes: https://github.com/OCA/sale-promotion/pull/166

Pending from:
- [ ] https://github.com/OCA/sale-promotion/pull/179

cc @Tecnativa TT44346

@chienandalu please review

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
You need to fix the temp deps commit

**Comment by pilarvargas-tecnativa** [comment]
ping @chienandalu

File: sale_loyalty_incompatibility/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Loyalty incompatibilities",
```
Comment (chienandalu) [inline sale_loyalty_incompatibility/__manifest__.py]:
```suggestion
    "name": "Loyalty incompatibilities in sales",
```

File: sale_loyalty_incompatibility/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#, python-format
+msgid ""
+"This promotion is incompatible with other set already in the order so it "
+"can't be applied."
+msgstr ""
+"Esta promoción es incompatible con otra ya presente en la orden por lo que "
+"no se puede aplicar."
+
+#~ msgid "Coupon Program"
+#~ msgstr "Programa de cupones"
```
Comment (chienandalu) [inline sale_loyalty_incompatibility/i18n/es.po]:
```suggestion
```

File: sale_loyalty_incompatibility/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "summary": "Allows to set incompatibility rules between promotions",
```
Comment (chienandalu) [inline sale_loyalty_incompatibility/__manifest__.py]:
```suggestion
    "summary": "Allows to set incompatibility rules between promotions in sale orders",
```

**Review by chienandalu — APPROVED** [review]
Tested :+1: Some formal comments

File: sale_loyalty_incompatibility/__manifest__.py (hunk 3 above)
Comment (pilarvargas-tecnativa) [reply sale_loyalty_incompatibility/__manifest__.py]: thanks :)

**Comment by pilarvargas-tecnativa** [comment]
@CarlosRoca13 can you review please?

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by pilarvargas-tecnativa** [comment]
ping @pedrobaeza :)

**Pull request merged**